docs/readthedocs/source/doc/Nano/Overview/tensorflow_inference.md

```diff
@@ -1,98 +1,225 @@
+
+    - INC (Intel Neural Compressor): ``pip install neural-compressor``
+
+    - OpenVINO: ``pip install openvino-dev``
+
+    - ONNXRuntime: ``pip install onnx onnxruntime onnxruntime-extensions tf2onnx neural-compressor``
+
+    We recommand installing all dependencies by ``pip install bigdl-nano[tensorflow,inference]``, because you may run into version issues if you install dependencies manually.
 ```
-**Quantization without extra accelerator**
 
-Without extra accelerators, `Model.quantize()` returns a Keras module with desired precision and accuracy. Taking MobileNetV2 as an example, you can add quantization as below:
+## Runtime Acceleration
+
+All available runtime accelerations are integrated in `InferenceOptimizer.trace(accelerator='onnxruntime'/'openvino')` with different accelerator values.
+
+Taking MobileNetV2 as an example, you can use runtime acceleration as below:
+
 ```python
 import tensorflow as tf
 from tensorflow.keras.applications.mobilenet_v2 import MobileNetV2
 import numpy as np
-from bigdl.nano.tf.keras import Model
+from bigdl.nano.tf.keras import Model, InferenceOptimizer
 
 # step 1: create your model
 model = MobileNetV2(weights=None, input_shape=[40, 40, 3], classes=10)
 model = Model(inputs=model.inputs, outputs=model.outputs)
 
-# step 2: prepare your data and dataloader
+# step 2: prepare your data and dataset
 train_examples = np.random.random((100, 40, 40, 3))
 train_labels = np.random.randint(0, 10, size=(100,))
 train_dataset = tf.data.Dataset.from_tensor_slices((train_examples, train_labels))
 
 # (Optional) step 3: Something else, like training ...
 model.fit(train_dataset)
 
-# step 4: execute quantization
-q_model = model.quantize(calib_dataset=train_dataset)
+# step 4: execute quantization using `OpenVINO` acceleration
+traced_model = InferenceOptimizer.trace(model, accelerator="openvino")
+# or setp 4: execute quantization using `ONNXRuntime` acceleration
+traced_model = InferenceOptimizer.trace(model, accelerator="onnxruntime")
 
 # run simple prediction
-y_hat = q_model(train_examples)
+y_hat = traced_model(train_examples)
 
-# evaluate, predict also support acceleration
-q_model.evaluate(train_dataset)
+# predict also support acceleration
+traced_model.predict(train_dataset)
+```
+
+## Quantization
+
+Quantization is widely used to compress models to a lower precision, which not only reduces the model size but also accelerates inference. BigDL-Nano provides `InferenceOptimizer.quantize()` API for users to quickly obtain a quantized model with accuracy control by specifying a few arguments. `Sequential` has similar usage, so we will only show how to use an instance of `Model` to enable quantization pipeline here.
+
+To use INC as your quantization engine, you can choose `accelerator=None/'onnxruntime'`. Otherwise, `accelerator='openvino'` means using OpenVINO POT (Post-training Optimization) to do quantization.
+
+### Quantization without extra accelerator
```

Comment:
The title here is strange for me. Seems like we are giving example in this section for both with/without extra acceleration?

Reply:
this is a typo, it should be "Quantization without Accuracy Control"